ortools/sat/csharp/CpModel.cs

```diff
@@ -615,6 +615,11 @@ public void AddHint(IntVar var, long value)
             model_.SolutionHint.Values.Add(value);
         }
 
+        public void AddAssumption(IntVar var)
```

Comment:
Wrong type. Maybe we should accept a iLiteral, and make sure the literal is positive.